examples/03_connectivity/plot_multi_subject_connectome.py

```diff
@@ -92,6 +94,8 @@ def plot_matrices(cov, prec, title):
 # -------------------
 atlas_imgs = image.iter_img(msdl_atlas_dataset.maps)
 atlas_region_coords = [plotting.find_xyz_cut_coords(img) for img in atlas_imgs]
+labels = msdl_atlas_dataset.labels
+labels = [label.decode("utf-8") for label in labels]
```

Comment:
That's ugly :$. Do you think that we should be doing the decoding in the fetch_atlas_msdl function?